provider/cmd/pulumi-resource-nuage/nuage_provider/container_function.py

```diff
@@ -335,6 +229,18 @@ def __init__(
         else:
             outputs["url"] = None
 
+        # Untag ecs urls and keep only {pulumi.get_organization()}:{resource_name}.
+        # rsplit is because the generated name contains suffix at the end and we also need to untag image without that suffix
+        args.image_uri.apply(
+            lambda generated_image_name: (
+                local.Command(
+                    resource_name,
+                    create=f"docker rmi {generated_image_name.rsplit('-', 1)[0]} && docker rmi {generated_image_name}",
```

Comment:
If we changed our suffix separator then we'd have to do it here as well 

It works but it's not perfect

Reply:
Oooh, even more tricky 

Okay, well that's why we have unit tests 🙂

Are we testing for this ? In case a regression comes from Pulumi docker ?